go/vt/tabletserver/query_executor_test.go

```diff
@@ -1112,7 +1112,7 @@ func newTransaction(tsv *TabletServer) int64 {
 }
```

Comment:
Same thing, we'll fix in a different PR. These checks didn't use to exist when we started, but we usually respect them in new code.